pkg/pool-manager/pool_test.go

```diff
@@ -51,13 +51,13 @@ var _ = Describe("Pool", func() {
 	createPoolManager := func(startMacAddr, endMacAddr string, fakeObjectsForClient ...runtime.Object) *PoolManager {
 		fakeClient := fake.NewSimpleClientset(fakeObjectsForClient...)
 		startPoolRangeEnv, err := net.ParseMAC(startMacAddr)
-		Expect(err).ToNot(HaveOccurred(), "should successfully parse starting mac address range")
+		Expect(err).To(Succeed(), "should successfully parse starting mac address range")
```

Comment:
Humm, feels better to use "Success" when you pass the cmd to the Expect in cases when there is just the error returned, it will be read like "this command shoud succeed"
```golang
Expect( net.ParseMAC(startMacAddr)).To(Succeed(), "should successfully parse starting mac address range")
```

But in the case that we store the error at an "err" variable feels better to talk about error ocurring, since an error can occurr or don't occurr but but they don't succeed

Reply:
I agree. will remove.

Reply:
I mean that if the function return just error then we can use Succeed otherwise we use HaveOcurred
```golang
func Foo() error...
func Bar() string, error...
Expect(Foo()).To(Succeed())
bar, err := Bar()
Expect(err).ToNot(HaveOcurred())
```